### PR TITLE
updates

### PR DESCRIPTION
**Pull request by irisslee**
1. Trimmed to two functions: all fires YTD and active fires  
2. Added github action to scrape
3. Updated README and converted to markdown

### CODE REVIEW

**Comment by palewire** [comment]
I might be wrong, but I think that for the README to fully convert to markdown you need to remove the indented `::` style code snippet introductions that ReST uses with the unindented ``` style that markdown uses

**Comment by irisslee** [comment]
ok changed it!

File: setup.py [hunk 1]
```diff
@@ -11,14 +11,13 @@ def read(fname):
     name='calfire-wildfires',
     version='0.0.4',
     description="Download wildfires data from CalFire",
-    long_description=read('README.rst'),
+    long_description=read('README.md'),
     author='Los Angeles Times Data Desk',
```
Comment (palewire) [inline setup.py:16]: One very nerdy trick is for Markdown to work with PyPI you have to set the long_description_content_type as see [here](https://github.com/palewire/nws-wwa/blob/main/setup.py#L15)

Comment (irisslee) [reply setup.py:16]: Added!

**Review by palewire — APPROVED** [review]

**Pull request merged**